Wiki.md

```diff
@@ -0,0 +1,117 @@
+## Intro
+[Error Prone](https://github.com/google/error-prone) check to detect missing AutoDispose scope within defined scoped elements.
```

Comment:
This isn't a complete sentence

Reply:
https://github.com/uber/AutoDispose/pull/162/commits/dcd50f8516c9fb63193e7fe724b345998ac81d0b

Reply:
https://github.com/uber/AutoDispose/pull/162/commits/dcd50f8516c9fb63193e7fe724b345998ac81d0b